File: root/common/src/main/java/org/sc/common/rest/controller/AccessibilityResponse.java
package org.sc.common.rest.controller;

import java.util.Collections;
import java.util.List;

public class AccessibilityResponse extends RESTResponse {
    List<AccessibilityNotification> accessibilityNotifications;

    public AccessibilityResponse(){}

    public AccessibilityResponse(List<AccessibilityNotification> accessibilityNotifications) {
        super(Status.OK, Collections.emptySet());
        this.accessibilityNotifications = accessibilityNotifications;
    }

    public List<AccessibilityNotification> getAccessibilityNotifications() {
        return accessibilityNotifications;
    }
}

File: root/common/src/main/java/org/sc/common/rest/response/AccessibilityResponse.kt
package org.sc.common.rest.response

import org.sc.common.rest.AccessibilityNotificationDto
import org.sc.common.rest.Status

data class AccessibilityResponse (val status: Status,
                                  val messages: Set<String>,
                                  val content : List<AccessibilityNotificationDto>) :
    RESTResponse()